CHANGELOG.md

```diff
@@ -4,6 +4,98 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
+## 0.15.0 - 2019-03-25
+
+### Added
+
+- We now initialize TLS certificates for etcd metrics over TCP on port
+  9979.  We also store etcd certificates in various `kube-system`
+  Secrets and ConfigMaps.
+- The Kubernetes client is now extracted from the release image's
+  `kube-client-agent` reference, replacing
+  `quay.io/coreos/kube-client-agent`.
+- On the control-plane and compute nodes, CRI-O's pause image is now
+  extracted from the release image's `pod` reference.
+- Initial work for user-provided infrastructure on AWS and vSphere,
+  including a new `user-provided-infrastructure` subcommand.
+
+### Changed
+
+- The install-config version has been bumped to `v1beta4` for changes
+  to more closely align with `Network.config.openshift.io`:
+    - `serviceCIDR` is now `serviceNetwork`.
+    - `clusterNetworks` is now `clusterNetwork`.
+    - `type` is now `networkType`.
+    - `hostSubnetLength` is now `hostPrefix`.
+    `v1beta3` is deprecated and will be removed in a future release.
+- On AWS and OpenStack, ports 9000 through 9999 are now open for UDP.
+  They had been open for TCP since 0.4.0, with a bugfix for 9990 ->
+  9999 in 0.13.0.
+- On AWS, we now create network interfaces for the control-plane
+  nodes explicitly, which allows for faster resource-creation time by
+  allowing greater parallelization.
+- On AWS, we now ask the machine-API operator to use
+  `aws-cloud-credentials` (created by [the credential
+  operator][credential-operator]) to fulfill our Machine(Set)s.
+- On OpenStack, resources are prefixed with the cluster ID to avoid
+  conflicts when running multiple clusters under the same tenant.
+- On OpenStack, machines are now configured with hostnames to allow
+  inter-VM communication.
+- On OpenStack, machines are now configured with default DNS
+  nameservers (1.1.1.1 and 208.67.222.222).
+- Several doc and internal cleanups.
+
+### Fixed
+
+- On AWS, the credentials-checking logic now allows root credentials,
+  although it logs a warning because this approach is not recommended.
+- On AWS, we only consider available zones when calculating defaults.
+  This reduces the chance of errors from attempting resource creation
+  in impaired or unavailable zones, although there's still a
+  possibility for a zone going unavailable after our check but before
+  resource creation.
+- On AWS, the bootstrap machine is now created in the first public
+  subnet, restoring SSH and journald access, and fixing a bug from
+  0.14.0.
+- On AWS, the Kubernetes API load balancers now use `/readyz` instead
+  of `/healthz` for health checks, which allows for more graceful
+  control-plane rotation.
+- On AWS, `destroy cluster` has some fixes for:
+    - Removing snapshots associated with copied AMIs, fixing a bug
+      from 0.14.0.
+    - Network-interface deletion, where we now remove all network
```

Comment:
Not sure if AWS docs uses hyphen. https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/using-eni.html

Reply:
> Not sure if AWS docs uses hyphen.

Fixed with 620c0c0f6 -> bb387148c.